.travis.yml

```diff
@@ -12,16 +11,19 @@ php:
 matrix:
   allow_failures:
     - php: hhvm
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS="--prefer-lowest"
 
 install:
   - eval `ssh-agent -s`
   - test/travis/setup-secure-shell.sh
   - composer self-update
-  - composer install --no-interaction --prefer-source
```

Comment:
The commit message specifies: The new Symfony Components v3 dependencies are written to the lock-file and thus would be installed on the travis PHP 5.4 build as well. Since they depend on PHP 5.5, the composer install command would fail.

Since PHP 5.4 needs to be kept - the locked composer dependencies need to be downgraded on travis build, which needs the added composer flags.